corehq/apps/custom_data_fields/templates/custom_data_fields/custom_data_fields.html

```diff
@@ -183,10 +183,27 @@
                 </div>
             </div>
         </fieldset>
-
+        {% if show_purge_existing %}
+        <fieldset>
+            <div class="col-sm-10 col-sm-offset-2">
+                <div class="checkbox">
+                    <label>
+                        <input name="remove-existing" type="checkbox" data-bind="checked: purge_existing">{% trans "Remove unused fields from existing data."%}
+                        <span class="hq-help-template"
+                              data-title="{% trans "Remove Unused Fields" %}"
+                              data-content="{% blocktrans with entity_string=view.entity_string|lower %}
+                                  If checked, this will remove any extra information from any existing {{ entity_string }}
+                                  that has data fields that are no longer used.
```

Comment:
i struggled a bit with this text and being too lazy to add a pluralized version of `{{ entity_string }}` to the template. open to any improvements.